include/deal.II/matrix_free/operators.h

```diff
@@ -834,6 +939,183 @@ namespace MatrixFreeOperators
       }
   }
 
+
+  //-----------------------------LaplaceOperator----------------------------------
+
+  template <int dim, int fe_degree, int n_q_points_1d, int n_components, typename Number>
+  LaplaceOperator<dim, fe_degree, n_q_points_1d, n_components, Number>::
+  LaplaceOperator ()
+    :
+    Base<dim, Number>(),
+    scalar_coefficient(NULL)
```

Comment:
A shared pointer is default-constructed to the zero pointer, so no need put that here.